### ISSUE TITLE
Feature request: keep a number of old versions of wsl-genie

### ISSUE THREAD

**Issue by Elberet** [arkane-systems/wsl-transdebian#16]
Please maintain a set of older versions of genie in this repository, rather than only including the latest and greatest version.

This change would be helpful, because - for example:

1. A recent change in genie 2.4 has broken genie on (my) Windows 10 system (ref: arkane-systems/genie#287). I could easily work around this issue by installing genie 2.3, but since that version is no longer in the package repository, rolling back to 2.3 is needlessly complicated.
2. In my use-case, WSL environments are ephemeral: quickly disposed and quickly reinstalled via script. In order to maintain reproducability of these environments, it is ideal to pin installed packages to know a known version, and only upgrade after reviewing a new release. This is impossible if the repository only ever contains the latest and greatest version, because the script would start failing as soon as a new genie release is published.

**Comment by cerebrate** [comment]
`reprepro` doesn't normally allow that for the same release (it automatically removes the old version whenever it adds a new version), which is going to add some extra complications anyway, but I'll see what can be done.

**Comment by cerebrate** [comment]
Okay, I've beaten my head against this for a while and concluded that apt just doesn't want to allow this; of all the tools to build apt repos, all but one (aptly) claims to only support one version at a time of any given package, and I can't get aptly to work usefully, so I'm afraid this one may go unanswered.

I don't suppose having the script `curl` the appropriate .deb file from the arkane-systems/genie [releases page](https://github.com/arkane-systems/genie/releases) and then install it with `sudo dpkg -i <filename>` is an option? That page should have installables for every version I release pretty much forever.

**Comment by Elberet** [comment]
That's what I'm doing right now. It would have been nicer to get all packages through the same type of mechanism, but oh well, so be it.

**Comment by cerebrate** [comment]
Yeah, it's pretty intensely frustrating. The apt client tools look like they should be able to handle, but none of the repo-side tooling will do it. I've asked a few questions of people more knowledgeable than me on the topic, but don't holdout all that much hope, I'm afraid.
